<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Ekran" name="ekran" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="verR">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>16</width>
      <height>131</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verG">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>16</width>
      <height>131</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verB">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>16</width>
      <height>131</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>150</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>150</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>150</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>360</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QSlider" name="verV">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>220</y>
      <width>16</width>
      <height>131</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>360</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>360</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QSlider" name="verH">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>220</y>
      <width>16</width>
      <height>131</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verS">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>220</y>
      <width>16</width>
      <height>131</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Ekran</class>
   <extends>QWidget</extends>
   <header>ekran.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
